corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -321,6 +344,13 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
                     usedWidth += elementTileWidth;
                 }
                 if (child.type === constants.GROUP_TYPE) {
+                    if (isNPerRowRepeatElement(child.style)) {
+                        for (let groupChild of child.children) {
+                            if (groupChild.type === constants.GROUP_TYPE) {
```

Comment:
Given what we learned [here](https://dimagi.atlassian.net/browse/QA-6043) about the type assigned to repeat groups in the formplayer response, I feel like this logic needs to change to handle that.

Reply:
applied grouping for `repeat-juncture` type elements https://github.com/dimagi/commcare-hq/pull/33979/commits/cd09d773b099ba55e0f762b59e2aa2a22ea17ae3